package ru.redenergy.itschool.task6;

import org.junit.Test;

public class Task6Test {

	@Test
	public void testZeroDivisionWarning() {
		Task6.main(); //if exception will be thrown test would fail
	}

}
